pkg/eval/builtin_fn_fs.go

```diff
@@ -84,13 +68,7 @@ func init() {
 		"dir-history": dirs,
 
 		// Path
-		"path-abs":      filepath.Abs,
```

Comment:
Do not remove existing builtins when moving the functionality to a new namespace. With this change they should be deprecated; not removed. That provides a transition period so as not to needlessly break existing users of the Elvish shell.